doc/adr/0013-marconi-query-interface.rst

```diff
@@ -0,0 +1,130 @@
+.. _marconi_indexer:
+
+ADR 13: Marconi Query Interface
+======================================================
+
+Date: 2022-10-27
+
+Author(s)
+---------
+
+Kayvan Kazeminejad <kayvan.kazeminejad@iohk.io>
+
+Status
+------
+
+Draft
+
+Context
+-------
+Marconi Query Interface is both an executable and a Haskell library built on top of `marconi`.  In its current alpha version, the interface provides a reporting capability for a bounded list of `Bech32 Shelley Era addresses <https://github.com/cardano-foundation/CIPs/tree/master/CIP-0019#shelley-addresses>`_ and their corresponding `UTXO` `TxOutRef`.  The executable package provides for an HTTP `JSON-RPC 2.0 <https://www.jsonrpc.org/specification>`_ on top of the marconi Haskell libraries.
+
+
+Scope
+-------
+
+The scope of this document is the marcoin query interface executable package built on top:
+
+- `marconi`
+- `marconi-query`
+- JSON-RPC 2.0
+- HTTP
+
+Caution
+^^^^^^
+
+This is a living document as the interface is under development
+
+
+Decision
+--------
+
+After soliciting a number of potential clients, we made the following architectural decisions:
```

Comment:
I suggest: 
After soliciting feedback from a number of ... 

Or maybe instead of "feedback" you may prefer "input" depending on what is most accurate.